src/Notification/OnNextObservableNotification.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Rx\Notification;
+
+use Rx\Observable;
+use Rx\Testing\ColdObservable;
+use Rx\Testing\HotObservable;
+use Rx\Testing\MockHigherOrderObserver;
+use Rx\Testing\TestScheduler;
+
+class OnNextObservableNotification extends OnNextNotification
```

Comment:
nit: should this live in `Rx\Testing` instead? It depends on the `TestScheduler`?